src/test/java/com/google/javascript/cl2dts/static_provided.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('goog.events.BrowserEvent');
+goog.provide('goog.events.BrowserEvent.MouseButton');
+/**
+ * @constructor
+ */
+goog.events.BrowserEvent = function() {
+};
+
+/**
+ * Normalized button constants for the mouse.
+ * @enum {number}
+ */
+goog.events.BrowserEvent.MouseButton = {
+  LEFT: 0,
+  MIDDLE: 1,
+  RIGHT: 2
+};
```

Comment:
Reworked a bit, now doing the Right Thing - PTAL

On Fri, Aug 28, 2015 at 4:22 PM Alex Eagle <alexeagle@google.com> wrote:

> Nope actually adding that exposes another bug. We get a duplicate def'n of
> the static method, inside the class as well as a namespace matching the
> class name. Needs more work.
>
> On Fri, Aug 28, 2015 at 3:33 PM Rado Kirov <notifications@github.com>
> wrote:
>
>> In src/test/java/com/google/javascript/cl2dts/static_provided.js
>> <https://github.com/angular/closure-to-dts/pull/58#discussion_r38250956>:
>>
>> > +goog.provide('goog.events.BrowserEvent.MouseButton');
>> > +/**
>> > + * @constructor
>> > + */
>> > +goog.events.BrowserEvent = function() {
>> > +};
>> > +
>> > +/**
>> > + * Normalized button constants for the mouse.
>> > + * @enum {number}
>> > + */
>> > +goog.events.BrowserEvent.MouseButton = {
>> > +  LEFT: 0,
>> > +  MIDDLE: 1,
>> > +  RIGHT: 2
>> > +};
>>
>> Since the test is called static_provided, what about adding a static
>> function that is also provided (I think it should just work).
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/angular/closure-to-dts/pull/58/files#r38250956>.
>>
>